### PR TITLE
Intro Post 

### PR DESCRIPTION
**Pull request by jesshami**
Adding intro post

### CODE REVIEW

File: _posts/2016-06-17-jess-intro.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Jess Hamilton Introduction"
```
Comment (arokem) [inline _posts/2016-06-17-jess-intro.md]: To be consistent with others: 
=> "Introducing DSSG Fellow Jess Hamilton"

**Comment by arokem** [comment]
I think there are two files in here. It seems that you had to change the date on the file-name? 

The most fool-proof way to rename a file and have git keep track of that is:

```
git mv fname1.md fname2.md
```

At this point, you can do something like: 

```
git rm fname1.md 
```

To remove the file you don't want to include.

File: _posts/2016-06-17-jess-intro.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+date:   2016-06-24
+author: Jess Hamilton
+---
+
+![An image]({{ site.url }}/assets/images/hamilton.jpg)
```
Comment (arokem) [inline _posts/2016-06-17-jess-intro.md]: You might want to copy the format used here: 

https://github.com/uwescience/DSSG2016/blame/gh-pages/_posts/2016-06-24-introducing-dssg-fellow-kara-woo.md#L8-L9

(setting the image width to be appropriate for your file). In particular, if you remove the line break between the insertion of the image and the next paragaph, the front page will contain both your photo and the first paragraph of your post, which looks slightly better than having just your photo.

**Comment by jesshami** [comment]
Sorry about that! Removed the extra file - also I used Kara's formatting in the up to date file.

**Comment by arokem** [comment]
Is it possible that you forgot to `git push` these last changes?

**Comment by jesshami** [comment]
Strange, I pushed after removing the file and everything said it was up to date in my terminal, but I could still see the file on github.  Probably not a best practice, but I just removed it from there. Sorry for all of the confusion thanks for bearing with me!

**Comment by arokem** [comment]
No worries -- all part of getting this off the ground. I believe this should work!

**Pull request merged**